pom.xml

```diff
@@ -29,4 +29,16 @@
 		<connection>scm:git:https://github.com/atteo/classindex.git</connection>
 		<url>https://github.com/atteo/classindex</url>
 	</scm>
+
+	<build>
+		<pluginManagement>
+			<plugins>
+                <plugin>
```

Comment:
This file uses tab for indentation, please change spaces to tabs for consistency

Reply:
You're right. Sorry. It's done.